rust/src/service/handle.rs

```diff
@@ -151,21 +87,28 @@ impl Handle {
     /// current round parameters.
```

Comment:
it would be helpful to know when the `None` case is returned e.g. "...when a round is not in progress".
similarly with `get_sum_dict` and `get_seed_dict`

Reply:
I added a disclaimer "the availability of the sum dictionary depends on the current state of the coordinator" but I'd rather not write anything too specific because that would require updating the documentation of each of these methods if the implementation changes. To the consumer of the service, the state of the coordinator is opaque with this implementation. Do you think we need expose it?

Reply:
could be useful possibly at a later date, but for now it's clear enough.